### ISSUE TITLE
Stake management page is a dead end... now what do i do?

### ISSUE THREAD

**Issue by GriffGreen** [1Hive/honey-pot#264]
I deposited... now tell me what i should do next!

<img width="1094" alt="Screen Shot 2021-03-07 at 9 51 09 AM" src="https://user-images.githubusercontent.com/14945613/110244079-64842500-7f5d-11eb-9f12-05fdc03be097.png">

**Comment by rperez89** [comment]
Hmm that's a good one, what else would you think might be useful here? because when you are creating a proposal it says to you that you need to stake in order to be able to create an action.

**Comment by GriffGreen** [comment]
Thats the problem... Here is my user journey:

i tried to make a proposal, it said i couldn't till i Staked 

so then I went to this page, and put in the deposit... 

then while waiting for the tx i saw a butterfly and forgot what i was even there to do.... so i called my mom and we laughed about how my brother is a mailman.

then I found myself on a staking page and was like... i didn't intend on staking in 1hive wtf am i doing on this page... 

and then I remembered, oh yeah i wanted to make a proposal and this was just means to an end... 

So I navigated back to the proposals page myself... eventually after getting lost in a couple pages 

Then I realized I still couldnt make the proposal and was like WTF! Bullshit! Your app doesn't work, I staked!!! 

Then i was like... oh well maybe the UX isnt perfected yet and i missed a step... AND I DID! That little tiny switch needed to be flicked (thats issue #263 ) 

So i flicked the switch and then while waiting for the 3 seconds to wait for the tx to go thru i started watching a youtube video and tweeting... and again forgot wtf i was doing... 

then i saw this staking page and was like... i don't give a fuck about staking, i didnt come here to do that... and closed the tab and forgot to make a proposal.

My point is... this page is not the final destination for any user journey... we should make sure that the user doesn't have to think, they came here to make a proposal, lets send them back to that page... Ideally with a pop up that tells them how incredible they are and has a button that takes them to where they want to be like #262
